repository-data/development/src/main/resources/hcm-content/content/documents/lks/guidance-pages.yaml

```diff
@@ -1,4 +1,5 @@
 /content/documents/lks/guidance-pages:
```

Comment:
@thojava - Seems like this document still has got some references to `hee:contentCardItem`

Reply:
@smanivasag I have updated both. Not sure why I can not see the error on my local

Reply:
Thanks @thojava